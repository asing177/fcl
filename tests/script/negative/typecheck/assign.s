@initial
init() {
  a = (b = 10); // rhs of assignment must be an expression (not 'void')
  terminate();
}
