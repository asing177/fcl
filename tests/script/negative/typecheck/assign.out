Error at Line 5, Column 12:
   Expected an expression on the right hand side of an assignment but got the statement 'b = 10'